Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentEnd/FocusHelper.js

```diff
@@ -0,0 +1,38 @@
+/* vim: set ts=2 sts=2 sw=2 et tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+const isButton = (element) => {
+  if (element.nodeName !== "INPUT") {
+    return false;
+  }
+
+  return (element.type == "BUTTON" || element.type == "SUBMIT");
```

Comment:
`nodeName` is definitely always capitalized, but I'm not sure about `type`. To be safe, you might want to check against `element.type.toUpperCase()`.

Reply:
It also can't hurt to check for `[type="file"]`, although when those are focused on iOS, the user is likely browsing the photo library.